### PR TITLE
Versioned base images of generatos

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

**Comment by rhuss** [comment]
Should be extracted in some base image, too.

File: generator/spring-boot/src/main/java/io/fabric8/maven/generator/springboot/SpringBootGenerator.java [hunk 1]
```diff
@@ -29,16 +29,20 @@
 import io.fabric8.maven.generator.api.MavenGeneratorContext;
 import io.fabric8.utils.Strings;
 import org.apache.maven.project.MavenProject;
+import org.apache.xbean.propertyeditor.IntegerEditor;
```
Comment (davsclaus) [inline generator/spring-boot/src/main/java/io/fabric8/maven/generator/springboot/SpringBootGenerator.java]: Is this the right class to import from xbeans?

Comment (rhuss) [inline generator/spring-boot/src/main/java/io/fabric8/maven/generator/springboot/SpringBootGenerator.java]: for sure not ;-)

thanks for the heads up ...

**Comment by rhuss** [comment]
lgtm

**Comment by rhuss** [comment]
[merge]

**Comment by rhuss** [comment]
lgtm

**Comment by rhuss** [comment]
[merge]

**Pull request merged**